src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const VeteranInformationComponent = props => {
+  let dateOfBirthFormatted = '';
+  let genderFull = '';
+  if (props.user && props.user.dob) {
+    dateOfBirthFormatted = moment(props.user.dob).format('MMMM Do YYYY');
+  }
+  if (props.user && props.user.gender === 'M') {
+    genderFull = 'Male';
+  } else if (props.user && props.user.gender === 'F') {
+    genderFull = 'Female';
```

Comment:
It could be worthwhile storing the strings 'Male' and 'Female' as constants somewhere.

Reply:
See https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/static-data/labels.jsx#L6

Reply:
I considered this but I figured it wasn't worth creating a new file just for two constants and they didn't make sense living in any of our other constants files

Reply:
Could you use the constants that @Mottie linked earlier? Looks like they already exist in platform and are ripe for the taking.